extensions/sql/transfer-process-store/ddl.sql

```diff
@@ -0,0 +1,84 @@
+create table lease
+(
+    leased_by      varchar               not null,
+    leased_at      bigint,
+    lease_duration integer default 60000 not null,
```

Comment:
I'm not sure this is the right time to set this default. If it's not already set in the lease, you're basically changing the object when persisting.

Reply:
Shouldn't we have defaults at every level? There should be defaults in Java code, at the DML level and at the DDL level, no?

Reply:
I'd prefer if they were maintained only at the level of the actual edc object. Otherwise a default might get added/changed/removed, while another default is hidden in the SQL implementation.

Reply:
fair enough. will remove.